app/components/data-source-menu.tsx

```diff
@@ -1,9 +1,10 @@
 import { Trans } from "@lingui/macro";
-import { MenuItem, Select, Typography } from "@mui/material";
+import { Typography } from "@mui/material";
 import { useRouter } from "next/router";
 import { useMemo } from "react";
 
 import Flex from "@/components/flex";
+import { Select } from "@/components/form";
```

Comment:
Nit: I wonder know if at some point it makes sense to use eslint rules to forbid use of MUI select inside app.

Reply:
I think it could make sense, but currently we have `Option` type which is of a string type and we have at least one custom select for color picker, to render color swatches instead of regular labels – but if we manage to consolidate them, then it would be nice 👍 I think in fact it could already work for less custom components like e.g. checkboxes or switches 🤔